### PR TITLE
Update tutorial notebooks and add custom RDF fitting demo

### PR DESCRIPTION
**Pull request by hmacdope**
## Description

Show how to use custom RDF fitting and tidy up tutorials.

## Todos
Notable points that this PR has either accomplished or will accomplish.
  - [X] Custom RDF fits
  - [X] Replace path based imports of trajectories with imports from `datafiles`

## Status
- [X] Ready to go

### CODE REVIEW

**Comment by hmacdope** [comment]
@orioncohen you okay to review?

File: tutorials/basics_tutorial.ipynb [hunk 1]
```diff
@@ -32,13 +32,14 @@
    "source": [
```
Comment (orioncohen) [inline tutorials/basics_tutorial.ipynb:32]: <p>It looks like the cell outputs from here onward are no longer displayed in ReviewNB. I could imagine the outputs would be useful for someone scanning the document and not wanting to run it themselves. Maybe run all the cells and recommit? Hopefully that will fix the issue.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/solvation-analysis/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

File: tutorials/rdf_fitting_demo.ipynb [hunk 2]
```diff
@@ -1,29 +1,76 @@
 {
```
Comment (orioncohen) [inline tutorials/rdf_fitting_demo.ipynb:1]: <p>This is great! I'm going to do some testing with scipy.find_peaks and see how it performs on the test examples I created. If it outperforms the current implementation, I'll use it as the default.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/solvation-analysis/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

**Comment by orioncohen** [comment]
> @orioncohen you okay to review?

Yep! This looks great! I'm excited that `scipy.find_peaks` was successful on such a noisy example. I have many rdfs saved in the test files. I'll see how it performs on those and likely make it the default.

Thanks for putting this together, I think this is an important addition.

File: tutorials/basics_tutorial.ipynb (hunk 1 above)
Comment (hmacdope) [reply tutorials/basics_tutorial.ipynb:32]: Done

**Comment by hmacdope** [comment]
@orioncohen if we feel like the default kernel could be switched to find-peaks, lets merge this first and make a note that we need to update the rdf fitting tutorial to reflect any possible change down the track.

**Comment by orioncohen** [comment]
Sweet this looks great! I made a note in issue #43. I'm going to go ahead and merge this now.

**Review by orioncohen — APPROVED** [review]
Looks good!

**Pull request merged**